### PR TITLE
Remove dyeingWeighting field & parameter.

### PR DESCRIPTION
**Pull request by n1k0**
[User Story](https://www.notion.so/Teinture-suppression-proc-d-Majorant-68b23837f673404381cdb2fdb4b5cb71)

### CODE REVIEW

**Comment by n1k0** [comment]
@paulboosz could you please look at the new scores and validates that they're ok for you? thanks!

File: src/Data/Textile/Formula.elm [hunk 1]
```diff
@@ -128,34 +128,22 @@ spinningImpacts impacts { spinningProcess, countryElecProcess } mass =
...
-    -> ( Process, Process ) -- Inbound: Dyeing processes (low, high)
-    -> Unit.Ratio -- Low/high dyeing process ratio
+    -> Process -- Inbound: Dyeing processes (low)
     -> Process -- Outbound: country heat impact
     -> Process -- Outbound: country electricity impact
     -> Mass
     -> { heat : Energy, kwh : Energy, impacts : Impacts }
-dyeingImpacts impacts ( dyeingLowProcess, dyeingHighProcess ) (Unit.Ratio highDyeingWeighting) heatProcess elecProcess baseMass =
+dyeingImpacts impacts dyeingProcess heatProcess elecProcess baseMass =
+    -- FIXME: rename dyingLowProcess to dyeingProcess
```
Comment (magopian) [inline src/Data/Textile/Formula.elm:137]: is this comment still relevant?

File: src/Data/Textile/Simulator.elm [hunk 2]
```diff
@@ -234,16 +234,15 @@ computeDyeingImpacts { processes } simulator =
     processes
         |> Process.loadWellKnown
         |> Result.map
-            (\{ dyeingHigh, dyeingLow } ->
+            (\{ dyeingLow } ->
```
Comment (magopian) [inline src/Data/Textile/Simulator.elm:237]: do you want to rename `dyeingLow` to `dyeing`?

**Review by magopian — APPROVED** [review]
That's a nice chunk of deleted code 👍 
A couple of very small nits.

**Comment by paulboosz** [comment]
Checked and coherent with gsheet ! Bravo !

**Pull request merged**